### DISCUSSION TITLE
augmenting a simple model with new training data

### DISCUSSION THREAD

**Discussion by rumi1359** [cornellius-gp/gpytorch#1869] [Q&A]
Lets say I've trained a ExactGPModel called model with some train_x and train_y tensors. Say I want to augment train_x with a new value: train_x = torch.cat((train_x, torch.tensor([x]))). 
_Question: I could reinitialize the model and train but is there a more efficient way?_

**Comment by wjmaddox** [comment]
This can be accomplished with `new_model = model.get_fantasy_model(new_x, new_y)`.
